packages/composite-checkout/src/types.ts

```diff
@@ -35,6 +35,7 @@ export interface PaymentMethod {
 	activeContent?: React.ReactNode;
 	inactiveContent?: React.ReactNode;
 	submitButton: ReactElement;
+	disabled?: boolean;
```

Comment:
I'm pretty sure this is not being used and can be removed.

Reply:
Removed.